test/javascripts/unit/components/query-result-test.js.es6

```diff
@@ -1,99 +1,94 @@
-import { moduleFor } from "ember-qunit";
-import { test } from "qunit";
+import {module, test} from "qunit";
+import {setupTest} from "ember-qunit";
```

Comment:
TIL: we need to configure the plugin linting action to run prettier on `/test` directory 😉

Reply:
Yep, I think I'll take a look now.

Reply:
Hmm, I'm pretty sure this was working before. Probably this https://github.com/discourse/discourse-data-explorer/pull/91 broke it but at first glance it runs prettier on `/test` directory 🤔.

Reply:
Looks like `**` in `ls test/**/*.@(js|es6)` doesn't traverse multiple levels of directories…
So it would have to be `ls test/javascripts/*/*.@(js|es6)` unless that somehow doesn't work for other plugins.

Reply:
Fixed in https://github.com/discourse/.github/pull/39 🙂